_includes/search.html

```diff
@@ -1,23 +1,44 @@
 <!-- HTML elements for search -->
+
+<!-- This is installed from the https://github.com/slashdotdash/jekyll-lunr-js-search Gem -->
+<script src="/js/search.min.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
This is a 138KB JS file. We should either put it right at the end of the body right before `</body>`.

If you do that, you should probably move the `$(...).lunrSearch` script to the bottom of the body also, because it depends on the 138KB bundle.

Reply:
done